src/element-service.js

```diff
@@ -63,6 +65,19 @@ export function getElementServiceIfAvailable(win, id, extension, opt_element) {
   if (s) {
     return /** @type {!Promise<?Object>} */ (s);
   }
+  /**
+   * If there is (or was) a stubbed extension wait for it to load before trying
+   * to get the service.  Prevents a race condition when everything but
+   * the extensions is in cache.  If there is no stub then it's either loaded,
+   * not present, or the service was defined by a test. In those cases
+   * we don't wait around for an extension that may not exist.
+   */
+  if (stubbedElementNames.includes(extension)) {
```

Comment:
`isElementScheduled` uses `win.ampExtendedElements` but I couldn't find anywhere in the code that actually gets set to anything other than empty.  This use case needs something that's populated when runtime scans the `<script>` tags in head and discovers custom elements.  Hooking elementStub is prefect because I don't get problems withe elements that are not extensions.

Reply:
NVM - it's aliased to knownElements ad set at the same time as stub, I'll switch to using `isElementScheduled`